code/modules/clothing/head/jobs.dm

```diff
@@ -81,36 +80,42 @@
 
 /obj/item/clothing/head/beret/sec/navy/hos
 	name = "Commander beret"
-	desc = "Grey beret with a Commander's rank emblem. For commanders that are more inclined towards style than safety."
+	desc = "Grey beret with a Leiutennant's rank emblem. For officers that are more inclined towards style than safety."
```

Comment:
Lieutenant is spelt wrong, might wanna change that

Reply:
fock